ui/@build/rollupProject/index.mjs

```diff
@@ -49,6 +49,7 @@ export default targets => {
         //   summaryOnly: true,
         // }),
       ],
+      onwarn: (warning, warn) => warning.code !== 'THIS_IS_UNDEFINED' && warn(warning),
```

Comment:
This silences an issue that I couldn't figure out how to resolve. I think globally silencing this kind of error isn't a big deal.

Alternatively, we could also do:

```suggestion
      context: 'this',
```

but I believe that changes the output and I wanted to avoid doing so.

Reply:
This doesn't seem like a warning we'd ever legitimately run into and would like to be warned about but probably still best to only ignore it in the module where it actually happens (ig via a rollup parameter).

For anybody else wondering, it happens in flatpickr which is imported by `site/src/tourForm.ts`. Here's the description of the error in the rollup docs: https://rollupjs.org/guide/en/#error-this-is-undefined

Reply:
Yeah exactly. I think there are 3 consumers of flatpickr IIRC, meaning a total of 4 places to modify. I can change it to ignore warnings in those 4 modules only, and we can always go back to something more global if we so desire. I'll make that change shortly

Reply:
Ok, updated in aba338b